src/utils/utils_req.c

```diff
@@ -319,7 +338,8 @@ void ocf_req_clear_map(struct ocf_request *req)
 			   sizeof(req->map[0]) * req->core_line_count, 0));
 }
 
+/* TODO: get rid of this and use async wait on counter  */
 uint32_t ocf_req_get_allocated(struct ocf_cache *cache)
 {
-	return env_atomic_read(&cache->pending_requests);
+	return env_atomic_read(&cache->refcnt.io_req.counter);
```

Comment:
I think it would be better to have ocf_refcnt_get() or similar getter instead of accessing refcnt fields directly. In general all operations on refcnt should be done using refcnt API, not using direct access.

Reply:
This code is removed in one of the following patches. The function to read current value is not needed anywhere in the code.